### ISSUE TITLE
add any model you want to use

### ISSUE THREAD

**Issue by sasha-droid** [benrugg/AI-Render#127]
### Describe the feature you'd like to see:

It would be cool to be able to add any model to use with SD. Thanks !

**Comment by benrugg** [comment]
You can do this in Automatic1111 already. Whatever model you select in the web ui will be used by AI Render. Is that what you're talking about, or something else?

**Comment by sasha-droid** [comment]
yes, i managed to do it by using stable diffusion locally. :) I was also wondering if it was possible to use animatediff with this addon ? thanks !

**Comment by benrugg** [comment]
Great. It doesn't work with AnimateDiff, although that would be cool.
